src/main/java/com/microsoft/lst_bench/util/StringUtils.java

```diff
@@ -75,4 +79,14 @@ public static FileExec replaceRegex(FileExec f, String regex, String replacement
                         s.getId(), pattern.matcher(s.getStatement()).replaceAll(replacement)))
             .collect(Collectors.toList()));
   }
+
+  public static String replaceEnvVars(File sourceFile) throws IOException {
+    if (sourceFile == null || !sourceFile.isFile()) {
+      // Nothing to do.
```

Comment:
Should there at least be a info message or warning?

Reply:
Done.

Reply:
Add the file path in case there is one for debugging?

Reply:
Done.